secure_fw/partitions/crypto/crypto_aead.c

```diff
@@ -30,7 +30,7 @@ psa_status_t tfm_crypto_aead_encrypt(psa_invec in_vec[],
 #else
     psa_status_t status = PSA_SUCCESS;
 
-    CRYPTO_IN_OUT_LEN_VALIDATE(in_len, 1, 3, out_len, 0, 1);
+    CRYPTO_IN_OUT_LEN_VALIDATE(in_len, 1, 3, out_len, 1, 1);
```

Comment:
This change makes the PSA API Crypto test number 224 fail.
It fails because the "Unsupported algorithm" test vector sets `ciphertext_size` to 0, and makes the test to fail too early.
Ideally, the test vector should be updated to have correct parameters, except for the unsupported algorithm.

Reply:
This was accidentally taken in while resolving merge conflicts. I have now removed it.